### PR TITLE
restart apache for jormun conf

### PR DESCRIPTION
**Pull request by Hoshiyo**
Need to be merge before prod deployment 2.25.0

### CODE REVIEW

File: fabfile/tasks.py [hunk 1]
```diff
@@ -181,6 +181,9 @@ def upgrade_all(up_tyr=True, up_confs=True, check_version=True, send_mail='no',
             execute(enable_all_nodes, env.eng_hosts, env.ws_hosts_1,  env.ws_hosts_2)
         env.roledefs['eng'] = env.eng_hosts
 
+    # restart for jormun
+    execute(jormungandr.reload_jormun_safe_all)
```
Comment (antoine-de) [inline fabfile/tasks.py]: they weren't reloaded at all ?

Comment (Hoshiyo) [inline fabfile/tasks.py]: not after jormun conf

Comment (antoine-de) [inline fabfile/tasks.py]: but there you are reloading it everytime no ?

File: fabfile/tasks.py [hunk 2]
```diff
@@ -177,9 +177,9 @@ def upgrade_all(up_tyr=True, up_confs=True, check_version=True, send_mail='no',
             instance = random.choice(env.instances.values())
             execute(jormungandr.test_jormungandr, get_host_addr(server), instance=instance.name)
 
-        if not manual_lb:
-            execute(enable_all_nodes, env.eng_hosts, env.ws_hosts_1,  env.ws_hosts_2)
-        env.roledefs['eng'] = env.eng_hosts
```
Comment (antoine-de) [inline fabfile/tasks.py]: :stuck_out_tongue_winking_eye: 

the enable_all_nodes reload the apaches ?

Comment (Hoshiyo) [inline fabfile/tasks.py]: restart ws2 and ws1 pool

**Pull request merged**